Src/ILGPU.Tests/StructureValues.cs

```diff
@@ -105,38 +53,12 @@ public void StructureInterop<T>(T value)
             using var buffer = Accelerator.Allocate1D<T>(1);
             Execute<Index1D, T>(buffer.IntExtent, buffer.View, value);
 
-            var expected = new T[] { value };
+            var expected = new T[] {value};
             Verify(buffer.View, expected);
         }
 
         public static TheoryData<object> StructureViewInteropData =>
-            new TheoryData<object>
-        {
-            { default(EmptyStruct) },
-            { default(TestStruct) },
-            { new TestStruct<byte>()
-            {
-                Val0 = 1,
-                Val1 = byte.MaxValue,
-                Val2 = short.MinValue
-            } },
-            { new TestStruct<int, TestStruct<float, TestStruct<EmptyStruct, sbyte>>>()
-            {
-                Val0 = int.MinValue,
-                Val1 = ushort.MaxValue,
-                Val2 = new TestStruct<float, TestStruct<EmptyStruct, sbyte>>()
-                {
-                    Val0 = float.Epsilon,
-                    Val1 = 0,
-                    Val2 = new TestStruct<EmptyStruct, sbyte>()
-                    {
-                        Val0 = default,
-                        Val1 = ushort.MinValue,
-                        Val2 = sbyte.MinValue,
-                    }
-                }
-            } }
-        };
+            new TheoryData<object> {{default(EmptyStruct)}, {default(TestStruct)}, {new TestStruct<byte>() {Val0 = 1, Val1 = byte.MaxValue, Val2 = short.MinValue}}, {new TestStruct<int, TestStruct<float, TestStruct<EmptyStruct, sbyte>>>() {Val0 = int.MinValue, Val1 = ushort.MaxValue, Val2 = new TestStruct<float, TestStruct<EmptyStruct, sbyte>>() {Val0 = float.Epsilon, Val1 = 0, Val2 = new TestStruct<EmptyStruct, sbyte>() {Val0 = default, Val1 = ushort.MinValue, Val2 = sbyte.MinValue,}}}}};
```

Comment:
QUESTION: The code formatter likes to collapse certain blocks, like this one. Are are able to prevent this?

Reply:
Could you please re-view this behavior? 👀 
It's not ideal, but at least it's not in one-line